File: Funciones/16. Contador_while/js/contador.js
/**
 * Contar hasta 5 con el ciclo while
 * Autor: Santiago GM
 * Fecha: 03/04/2024
 */

let numero;

function contar(pnumero) {
    numero = pnumero;
    let contador = 0;
    let resultado = "";

    while (contador < numero) {
        contador = contador + 1;
        resultado +=  contador + "\n"; 
    }

    return resultado; 
}

File: python/16. Contador_while/utils/contador.py
# Contar hasta 5 con el ciclo while
# Autor: Santiago GM

#Como Paramétros

def contar_hasta_5(numero_p) :
    numero, contador, resultado = numero_p, 0, ''

    while (contador < numero) :
        contador = contador + 1
        resultado +=  f'{contador}\n'
    return resultado